modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -14,48 +14,53 @@ endif::[]
+Default policies that internally enforce client application credentials include:
 
 * xref:rate-limiting-and-throttling-sla-based-policies.adoc[Rate Limiting - SLA-Based Policies]
-
++
 In Mule 3.8.5 or later, the following OAuth 2.0 Token Enforcement policies include a client ID enforcement validation:
-
 * xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 Access Token Enforcement]
 * xref:policy-ping-federate.adoc[PingFederate OAuth 2.0 Token Enforcement]
 * xref:policy-openid-connect.adoc[OpenID Connect OAuth 2.0 Token Enforcement]
+* xref:policy-mule4-jwt-validation.adoc[JWT Validation]
 
-= Offline mode
+Although token enforcement policies can optionally skip client validation, MuleSoft recommends that you enforce the validation to ensure that the token is associated to an approved contract.
 
-Credentials data from client application credentials registered to protected APIs, are provided by Anypoint Platform. Once a policy requiring such information is applied (eg Client ID enforcement policy), the Runtime caches persistently the information locally updating it periodically.
+== How the Policy Works
 
-Besides its performance benefits, this action enables the policy to work offline in the event of a sudden loss of connectivity to the platform. The enforcement is applied regardless of the state of the management plane, with existing data. Once connectivity is re-established, the cache will be periodically updated again.
+Every client application that wants to consume an API protected by a Client ID enforcement policy needs to have previously requested access to the API. After an approved contract exists between the application and the API, every request needs to include the client credentials in compliance with how the policy has been configured.
 
-= REST Console
-To prevent user requests from being rejected when using the REST Console, create a trait in the RAML root and then reference it in every operation of your API affected by policy. You may find the RAML snippet link containing the RAML code you need to add to the RAML in the corresponding policy from the list of applied policies in the Policies tab of your API specification in API Manager.
+For example, if the policy has been configured to expect a client ID and client secret as headers, the application needs to make sure that those credentials are sent in the request using the corresponding headers.
 
 image::raml-snippet.png[]
 
-= Credentials source
+== Configure the Policy
+
+You can configure the Client ID Enforcement policy to obtain the client credentials in two different ways, HTTP basic authentication header and custom expression. 
+
+*HTTP Basic Authentication Header*
+
+You can configure the policy to require credentials as part of the Authorization header by selecting the HTTP Basic Authentication option. In this case, the application consuming the API must use the basic authentication to send the credentials in the requests.
```

Comment:
replace "the basic authentication" with "the basic authentication scheme"

Reply:
Done! Thanks!